protobuf/exec.proto

```diff
@@ -19,10 +19,28 @@ option (gogoproto.sizer_all) = true;
 option (gogoproto.goproto_registration) = true;
 option (gogoproto.messagename_all) = true;
 
+message BlockEvent {
```

Comment:
Now we have BlockExecution and BlockEvent which both contain TxExecutions.

How about naming BlockEvent -> BlockExecutionStream or StreamingBlockExecution?